ProyectoFinal/src/visual/RegEmpresa.java

```diff
@@ -61,48 +69,162 @@ public RegEmpresa() {
 		panel.setLayout(null);
 
 		JLabel lblNewLabel = new JLabel("RNC:");
-		lblNewLabel.setBounds(71, 29, 136, 14);
+		lblNewLabel.setBounds(33, 32, 94, 14);
 		panel.add(lblNewLabel);
 		try{
 			txtFCedCliente = new JFormattedTextField(Utils.getMaskCedula());
 			txtFCedCliente.setForeground(Color.BLACK);
 			txtFCedCliente.setToolTipText("");
-			txtFCedCliente.setBounds(198, 25, 156, 20);
+			txtFCedCliente.setBounds(100, 29, 156, 20);
 			panel.add(txtFCedCliente);
 		}catch(Exception ex){}
 		{
 			JLabel lblNewLabel_1 = new JLabel("Tipo de empresa:");
-			lblNewLabel_1.setBounds(71, 118, 98, 14);
+			lblNewLabel_1.setBounds(382, 73, 98, 14);
 			panel.add(lblNewLabel_1);
 		}
 		{
 		    cbxTipo = new JComboBox();
-			cbxTipo.setModel(new DefaultComboBoxModel(new String[] {"<Seleccione>", "P\u00FAblica", "Privada", "Semiprivada"}));
-			cbxTipo.setBounds(198, 117, 156, 20);
+			cbxTipo.setModel(new DefaultComboBoxModel(new String[] {"<Seleccione>", "Micro", "Mediana", "Peque\u00F1a ", "Grande", "Emergente"}));
+			cbxTipo.setBounds(500, 70, 148, 20);
 			panel.add(cbxTipo);
 		}
 		{
 			JLabel lblNewLabel_2 = new JLabel("Sector:");
-			lblNewLabel_2.setBounds(71, 165, 98, 14);
+			lblNewLabel_2.setBounds(33, 73, 98, 14);
 			panel.add(lblNewLabel_2);
 		}
 		{
-			JLabel lblNombre = new JLabel("Nombre:");
-			lblNombre.setBounds(71, 69, 98, 14);
+			JLabel lblNombre = new JLabel("Nombre Comercial:");
+			lblNombre.setBounds(382, 32, 141, 14);
 			panel.add(lblNombre);
 		}
 		{
 			txtCodigo = new JTextField();
 			txtCodigo.setColumns(10);
-			txtCodigo.setBounds(197, 65, 156, 20);
+			txtCodigo.setBounds(100, 112, 156, 20);
 			panel.add(txtCodigo);
 		}			
 		{
 			textField = new JTextField();
 			textField.setColumns(10);
-			textField.setBounds(198, 159, 156, 20);
+			textField.setBounds(500, 29, 148, 20);
 			panel.add(textField);
 		}
+		{
+			JComboBox comboBox = new JComboBox();
+			comboBox.setModel(new DefaultComboBoxModel(new String[] {"<Seleccione>", "Industrial", "Agricultura ", "Alimentaci\u00F3n", "Comercio ", "Construcci\u00F3n", "Educaci\u00F3n ", "Hoteler\u00EDa", "Medios de comunicaci\u00F3n ", "Miner\u00EDa ", "Petrolero ", "Telecomunicaciones ", "Salud ", "Financieros", "P\u00FAblico ", "Silvicultura ", "Textil ", "Tecnol\u00F3gico", "Transporte"}));

```

Comment:
```suggestion
			comboBox.setModel(new DefaultComboBoxModel(new String[] {"<Seleccione>", "Industrial", "Agricultura", "Alimentaci\u00F3n", "Comercio", "Construcci\u00F3n", "Educaci\u00F3n", "Hoteler\u00EDa", "Medios de comunicaci\u00F3n", "Miner\u00EDa", "Petrolero", "Telecomunicaciones", "Salud", "Financieros", "P\u00FAblico", "Silvicultura", "Textil", "Tecnol\u00F3gico", "Transporte"}));

```

Hay espacios en los datos.